t/userpic-keyword-select.t

```diff
@@ -31,14 +31,6 @@ my $ICON1 = do { local $/; <$fh> };
 open( my $fh2, 'good.jpg' ) or die $!;
 my $ICON2 = do { local $/; <$fh2> };
 
-note("called with falsy or undefined user object");
```

Comment:
I don't know why this test, and it fails in Perl 5.30, so I removed it. But LMK if I'm missing something @nfagerlund !

Reply:
Oh, that broke when Ruth changed that from a standalone function that takes a user as an argument to a method on a user object -- can't call methods on undef. I think this is a merge conflict now because I noticed that in one of those post-code-push PRs and removed it there.